reana_server/rest/workflows.py

```diff
@@ -349,6 +356,7 @@ def get_workflows(user, **kwargs):  # noqa
             sort=sort,
             status=status or None,
             verbose=bool(verbose),
+            include_last_command=include_last_command or verbose,
```

Comment:
I don't think this is needed, as it's part of `kwargs`. See for example how `include_workspace_size` is not passed explicitly. Am I right?